modules/backend/formwidgets/TagList.php

```diff
@@ -208,7 +208,7 @@ protected function getSeparatorCharacter()
     {
         switch (strtolower($this->separator)) {
             case 'comma':
-                return ',';
+                return ', ';
```

Comment:
@huynhnamegal @LukeTowers If this is to be accepted, I think it should be a different option as opposed to changing the format of `comma` separated tags, so it doesn't break any current implementations.
```suggestion
                return ',';
            case 'spacedComma':
                return ', ';
```

Reply:
That's great! In my country, it's look great if we have a space after comma.

Reply:
@bennothommo haven't looked into this too deeply, how would it break existing setups?

Reply:
@LukeTowers It probably won't - I'm just guessing from the intention of having a separate result for comma or space. It may be that some people use the value of the TagList for comma-separated values.

Reply:
@huynhnamegal please implement the above suggestion and add a PR to the docs as well.